api/lib/block-listeners/polkadot-node-subscription.js

```diff
@@ -165,27 +165,24 @@ class PolkadotNodeSubscription {
           )
           this.currentEra = era
 
-          const rewardsScript = spawn(
-            'node',
-            [
-              'scripts/getOldPolkadotRewardEras.js',
-              `--currentEra=${this.currentEra}`
-            ],
-            {
-              stdio: 'inherit' //feed all child process logging into parent process
-            }
-          )
-          rewardsScript.on('close', function (code) {
-            process.stdout.write(
-              'rewardsScript finished with code ' + code + '\n'
+          try {
+            console.log(
+              'Starting Polkadot rewards script on',
+              config.scriptRunnerEndpoint
             )
-
-            if (code !== 0) {
-              Sentry.captureException(
-                new Error('getOldPolkadotRewardEras script failed')
-              )
-            }
-          })
+            fetch(`${config.scriptRunnerEndpoint}/polkadotrewards`, {
```

Comment:
```suggestion
            await fetch(`${config.scriptRunnerEndpoint}/polkadotrewards`, {
```

Reply:
why? we don't need to wait for this to finish